src/test/suite/language/mongoDBService.test.ts

```diff
@@ -590,6 +590,30 @@ suite('MongoDBService Test Suite', () => {
       expect(completion).to.have.property('detail');
     });
 
+    test('provide system variable completion in find', async () => {
+      const result = await testMongoDBService.provideCompletionItems(
+        'db.collection.find({ _id: "$$N" });',
+        { line: 0, character: 30 }
+      );
+      const completion = result.find(
+        (item: CompletionItem) => item.label === '$$NOW'
+      );
+
+      expect(completion).to.have.property('kind', CompletionItemKind.Variable);
+    });
+
+    test('provide system variable completion in aggregate', async () => {
+      const result = await testMongoDBService.provideCompletionItems(
+        'db.collection.aggregate([{ $match: { _id: "$$R" }}]);',
```

Comment:
Trying this manually I wasn't able to get it to work. Does it work for you?

Reply:
Do you trigger suggestions via `Ctrl+Space` inside the string? https://github.com/mongodb-js/vscode#additional-settings